src/FlatButtonConfirm.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import {
+    FlatButton,
+    RaisedButton,
+    Popover,
+} from 'material-ui/';
+
+import { ButtonGroup, ClearFix } from './utils';
+
+export default class PopoverExampleSimple extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      open: false,
+    };
+  }
+
+  handleTouchTap = (event) => {
+    // This prevents ghost click.
+    event.preventDefault();
+
+    this.setState({
+      open: true,
+      anchorEl: event.currentTarget,
```

Comment:
This was taken from the MUI examples. I don't have much of an opinion here.

This component has issues and needs some attention before it can be used in prod so am holding off on documenting it. If you could take a look at it and submit a PR I would be super grateful.